Makefile

```diff
@@ -349,10 +349,12 @@ static: skydive.clean govendor genlocalfiles
 .PHONY: contribs.clean
 contribs.clean:
 	$(MAKE) -C contrib/snort clean
+	$(MAKE) -C contrib/objectstore clean
```

Comment:
Sorry a final remark, I think objectstore is too generic especially since the implementation is based on aws. I would use awss3 or something similar.

Reply:
@safchain It supports all S3-compliant object stores, which includes: IBM Cloud Object Storage and Google Cloud Storage.
Also, the s3-specific part of the code is modular, and can be easily extended to support other object storage APIs (via the `Client` interface).